<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="PengOst-MAC.local" ignored="0" name="Tests" tests="15" failures="4" timestamp="28 Dec 2020 10:12:21 GMT" time="63.069" errors="0">
  <testcase name="demoTest" time="0.001" classname="com.sample.test.demo.tests.DemoTest"/>
  <testcase name="NEG_boundary_value_test1" time="4.313" classname="com.sample.test.demo.tests.Functional_NEG">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.sample.test.demo.tests.Functional_NEG.NEG_boundary_value_test1(Functional_NEG.java:65)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- NEG_boundary_value_test1 -->
  <testcase name="Sanity_place_order_cash" time="8.443" classname="com.sample.test.demo.tests.SanityTest"/>
  <testcase name="REP_place_order_cash" time="5.129" classname="com.sample.test.demo.tests.Functional_POS"/>
  <testcase name="REP_place_order_cash_missing_field" time="5.988" classname="com.sample.test.demo.tests.Functional_POS"/>
  <testcase name="Sanity_place_order_cc" time="8.251" classname="com.sample.test.demo.tests.SanityTest"/>
  <testcase name="REP_place_order_cash_missing_nonmad_field" time="6.01" classname="com.sample.test.demo.tests.Functional_POS"/>
  <testcase name="NEG_boundary_value_test2" time="13.48" classname="com.sample.test.demo.tests.Functional_NEG">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.sample.test.demo.tests.Functional_NEG.NEG_boundary_value_test2(Functional_NEG.java:88)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- NEG_boundary_value_test2 -->
  <testcase name="NEG_select_both_payment" time="1.761" classname="com.sample.test.demo.tests.Functional_NEG">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.sample.test.demo.tests.Functional_NEG.NEG_select_both_payment(Functional_NEG.java:19)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- NEG_select_both_payment -->
  <testcase name="REP_place_order_cc" time="5.421" classname="com.sample.test.demo.tests.Functional_POS"/>
  <testcase name="NEG_wrong_field_information" time="5.066" classname="com.sample.test.demo.tests.Functional_NEG">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.sample.test.demo.tests.Functional_NEG.NEG_wrong_field_information(Functional_NEG.java:43)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- NEG_wrong_field_information -->
  <testcase name="REP_place_order_refresh_clean_field" time="4.814" classname="com.sample.test.demo.tests.Functional_POS"/>
  <testcase name="REP_place_order_reset_clean_field" time="2.221" classname="com.sample.test.demo.tests.Functional_POS"/>
  <testcase name="Sanity_place_order_cash" time="3.947" classname="com.sample.test.demo.tests.SanityTest"/>
  <testcase name="Sanity_place_order_cc" time="3.079" classname="com.sample.test.demo.tests.SanityTest"/>
</testsuite> <!-- Tests -->
